mkdocs.yml

```diff
@@ -32,6 +32,7 @@ pages:
 - Whitepapers:
     - Filesystem on SD card: 'sdcard.md'
 - C Modules:
+    - 'Writing external C modules' 'modules/extmods.md'
```

Comment:
I was just about to finally merge this when I spotted this line. We will possible settle on a different structure in the future but for now can you please move this to the section above, thanks. It's definitely not a module.

Reply:
I moved it under FAQs, next to Extension Developer FAQs. Does that work or you meant as "whitepaper"?

Reply:
Yes, I meant the latter.

Reply:
Done now. Thanks

Reply:
Thank you but it looks like you got the indentation wrong. That would produce a "FAQs" submenu.

Reply:
🤦🏻‍♂️ Sorry for all the trouble. I think I got it right now.